apiserver/facades/agent/proxyupdater/proxyupdater_test.go

```diff
@@ -53,13 +57,18 @@ func (s *ProxyUpdaterSuite) SetUpTest(c *gc.C) {
 	s.state.SetUp(c)
 	s.AddCleanup(func(_ *gc.C) { s.state.Kill() })
 
-	api, err := proxyupdater.NewAPIV2(s.state, s.state, &stubControllerConfigService{}, s.resources, s.authorizer)
+	s.modelConfigService = &stubModelConfigService{}
```

Comment:
Looking at the facade code, I think moving to mocks would be trivial. Can we please change to that?